src/pages/registration/contributors_tab/components/AuthorCard.tsx

```diff
@@ -52,99 +120,99 @@ const AuthorCard: FC<AuthorCardProps> = ({
   const [emailValue, setEmailValue] = useState(values.entityDescription.contributors[index]?.email ?? '');
 
   return (
-    <TableRow tabIndex={0} key={author.identity.id}>
-      <TableCell align="left">
-        <Field name={`${baseFieldName}.${SpecificContributorFieldNames.SEQUENCE}`}>
-          {({ field }: FieldProps) => (
-            <TextField
-              {...field}
-              type="number"
-              onKeyDown={(event) => {
-                if (event.key === 'Enter' && field.value) {
-                  event.preventDefault();
-                  onMoveCard(event);
-                }
-              }}
-              onBlur={(event) => {
-                onMoveCard(event);
-                field.onBlur(event);
-              }}
-            />
-          )}
-        </Field>
-        <Typography variant="h5">
-          {author.identity.name}{' '}
-          {author.identity.arpId ? (
-            <Tooltip title={t('contributors.known_author_identity') as string}>
-              <StyledCheckIcon />
-            </Tooltip>
-          ) : (
-            <Tooltip title={t('contributors.unknown_author_identity') as string}>
-              <StyledWarningIcon />
-            </Tooltip>
-          )}
-        </Typography>
-
-        <Field name={`${baseFieldName}.${SpecificContributorFieldNames.CORRESPONDING}`}>
-          {({ field }: FieldProps) => (
-            <FormControlLabel
-              data-testid="author-corresponding-checkbox"
-              control={<Checkbox checked={!!field.value} {...field} />}
-              label={t('contributors.corresponding')}
-            />
-          )}
-        </Field>
-        <div>
-          {author.correspondingAuthor && (
-            <Field name={`${baseFieldName}.${SpecificContributorFieldNames.EMAIL}`}>
-              {({ field, meta: { error, touched } }: FieldProps) => (
-                <StyledEmailTextField
-                  data-testid="author-email-input"
-                  variant="outlined"
-                  label={t('common:email')}
+    <StyledAuthorCard>
+      <StyledContent>
+        <StyledAuthorSection key={author.identity.id}>
+          <StyledNameField variant="h5">
+            <b>{author.identity.name}</b>
+          </StyledNameField>
+          <StyledVerifiedSection>
+            {author.identity.arpId ? (
+              <>
+                <Tooltip title={t('contributors.known_author_identity') as string}>
```

Comment:
stusset over `as string` tidligere en gang, og fant ut at `t<string>(...)` også kan brukes. bare sånn fyi, for det blir jo samme greia